src/archive/mod.rs

```diff
@@ -334,9 +335,9 @@ pub struct Archive<'a> {
     sysv_name_index: NameIndex<'a>,
     // the array of members, which are indexed by the members hash and symbol index
     member_array: Vec<Member<'a>>,
-    members: HashMap<&'a str, usize>,
+    members: BTreeMap<&'a str, usize>,
```

Comment:
Why did this change? Because of random requirements?

Reply:
Meh it’s fine. I don’t think O(n) lookup important for members, since there’s usually only like 10